Classes/Navigation/BreadcrumbsBuilder.php

```diff
@@ -99,7 +105,8 @@ public function buildBreadcrumbs(
                     if ($category !== null && !$category->isNavHide()) {
                         $breadcrumbs[] = [
                             'title' => $category->getAlternativeTitle() ?: $category->getTitle(),
-                            'uid' => $value,
+                            'product_uid' => $value,
```

Comment:
@mabolek Hey ) this change introduces regression bug. In method "buildLink" it checks if "uid" is set in order to build deep category url that includes parent categories like "parentcategory/subcategory/subsubcategory".

https://github.com/pixelant/pxa_product_manager/blob/e802734492070bdf426a3c43bd0432b941d575e1/Classes/Navigation/BreadcrumbsBuilder.php#L155

Reply:
Hi @anjeylink! Hope you're doing well! Thanks for pointing this out. Happy to see you're checking in to see how your baby is doing.